src/walk/index.js

```diff
@@ -335,8 +335,8 @@ base.NewExpression = base.CallExpression = (node, st, c) => {
       c(arg, st, "Expression")
 }
 base.MemberExpression = (node, st, c) => {
+  c(node.property, st, "Expression")
   c(node.object, st, "Expression")
-  if (node.computed) c(node.property, st, "Expression")
```

Comment:
Non-variable identifiers are intentionally skipped by default. The `foo` in 'x.foo`, though it is an identifier, is not a variable reference in the way that a regular identifier expression node would be. This can't be changed without seriously breaking code that relies on it.

Reply:
… and same for property keys, sure.